res/database/Default/database_news.xml

```diff
@@ -5616,93 +5770,72 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
 			<title>
-				<de>Straßenfeger findet 1.000 Mark</de>
-				<en>Street sweeper finds 1,000 DEM</en>
-				<pl>Zamiatarka uliczna znajduje 1 000 DEM</pl>
+				<de>Straßenfeger findet ${onethousanddem}</de>
+				<en>Street sweeper finds ${onethousanddem}</en>
+				<pl>Zamiatarka uliczna znajduje ${onethousanddem}</pl>
 			</title>
 			<description>
-				<de>Besser hätte der Morgen nicht anfangen können: Beim Arbeiten fand ein Straßenfeger in Hamburg 1.000 Mark. „Die wer'n sofort versofff'n“, so der Finder.</de>
-				<en>The morning couldn't have started better: While working, a street sweeper in Hamburg found 1,000 DEM. "I'm... I'mma booze 'em up rite awwway," said the finder.</en>
-				<pl>Poranek nie mógł zacząć się lepiej: Podczas pracy zamiatacz ulic w Hamburgu znalazł 1000 DEM. "Ja... zaraz je wypiję" - powiedział znalazca.</pl>
-			</description>
-			<effects>
-				<effect trigger="happen" type="triggernews" news="9c1f0f27-72d6-4bbd-b271-939ab31c3010_1" />
-			</effects>
-			<data genre="4" price="1.0" quality="19" />
-			<availability year_range_from="-1" year_range_to="2001" />
-		</news>
-		<news guid="9c1f0f27-72d6-4bbd-b271-939ab31c3010_1" thread_guid="baadc4ea-796b-44a2-b4cb-d333f01516ea_1" type="2" creator="">
-			<title>
-				<de>Straßenfeger tot</de>
-				<en>Street sweeper dead</en>
-				<pl>Zamiatacz ulic nie żyje</pl>
-			</title>
-			<description>
-				<de>Der Finder des 1.000-Mark-Scheins starb an einer Alkoholvergiftung. Auf seinem Grabstein steht: „...und er kehrt nie wieder!“.</de>
-				<en>The man who found the 1,000-mark bill died of alcohol poisoning. On his gravestone is written: "...and he'll never sweep through the docklands again".</en>
-				<pl>Mężczyzna, który znalazł banknot 1000-markowy, zmarł w wyniku zatrucia alkoholem. Na jego nagrobku widnieje napis: "...i już nigdy nie będzie przeczesywał doków".</pl>
+				<de>Besser hätte der Morgen nicht anfangen können: Beim Arbeiten fand ein Straßenfeger in Hamburg ${onethousanddem}. „Die wer'n sofort versofff'n“, so der Finder.</de>
+				<en>The morning couldn't have started better: While working, a street sweeper in Hamburg found ${onethousanddem}. "I'm... I'mma booze 'em up rite awwway," said the finder.</en>
+				<pl>Poranek nie mógł zacząć się lepiej: Podczas pracy zamiatacz ulic w Hamburgu znalazł ${onethousanddem}. "Ja... zaraz je wypiję" - powiedział znalazca.</pl>
 			</description>
+			<variables>
+				<currentyear>${.worldtime:"year"}</currentyear>
+				<onethousanddem>
+					<de>${.lte:currentyear:2001:"1.000 Mark":"500 Euro"}</de>
+					<en>${.lte:currentyear:2001:"1,000 DEM":"500 euro"}</en>
+					<pl>${.lte:currentyear:2001:"1 000 DEM":"500 euro"}</pl>
+				</onethousanddem>
+				<onethousandmarkbill>
+					<de>${.lte:currentyear:2001:"1.000-Mark-Scheins":"500-Euro-Scheins"}</de>
+					<en>${.lte:currentyear:2001:"1,000-mark bill:"500 euro bill"}</en>
```

Comment:
missing quotation mark after `mark bill` - before colon

Reply:
thanks, wondering why i didn't notice this in ronny's test tool, seems to still work.